### PR TITLE
Add license header to files under core/providers/cuda or contrib_ops/cuda/

### PR DESCRIPTION
**Pull request by tianleiwu**
### Description
Some files have no license header.

### CODE REVIEW

File: onnxruntime/core/providers/cuda/cu_inc/variadic_elementwise_impl.cuh [hunk 1]
```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```
Comment (tianleiwu) [inline onnxruntime/core/providers/cuda/cu_inc/variadic_elementwise_impl.cuh:2]: @edgchen1 to review

File: onnxruntime/core/providers/cuda/cuda_provider_interface.cc [hunk 2]
```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```
Comment (tianleiwu) [inline onnxruntime/core/providers/cuda/cuda_provider_interface.cc:2]: @guoyuhong to review

File: onnxruntime/core/providers/cuda/cupti_manager.cc [hunk 3]
```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```
Comment (tianleiwu) [inline onnxruntime/core/providers/cuda/cupti_manager.cc:2]: @abudup to review

File: onnxruntime/core/providers/cuda/tensor/resize_impl.cu [hunk 4]
```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```
Comment (tianleiwu) [inline onnxruntime/core/providers/cuda/tensor/resize_impl.cu:2]: @duli2012, @HectorSVC to review.

File: onnxruntime/core/providers/cuda/cuda_provider_interface.cc (hunk 2 above)
Comment (guoyuhong) [reply onnxruntime/core/providers/cuda/cuda_provider_interface.cc:2]: Good to know. Thanks~

**Review by guoyuhong — APPROVED** [review]

**Pull request merged**